### PR TITLE
Fixing builds

### PR DESCRIPTION
**Pull request by DaleCantwell**
Hard code to 8.x.x for iOS builds

### CODE REVIEW

**Comment by studentIvan** [comment]
Please solve it, we need this pr

File: intercom-plugin/scripts/checkForUpdate.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 function fetchUpdateInfo(context, callback) {
-  var fs = context.requireCordovaModule('fs');
+  var fs = context.require('fs');
```
Comment (piwel) [inline intercom-plugin/scripts/checkForUpdate.js]:
```suggestion
  var fs = require('fs');
```

File: intercom-plugin/scripts/checkForUpdate.js [hunk 2]
```diff
@@ -19,14 +19,14 @@ function fetchUpdateInfo(context, callback) {
 }
 
 function writeUpdateInfo(context, updateInfo, callback) {
-  var fs = context.requireCordovaModule('fs');
+  var fs = context.require('fs');
```
Comment (piwel) [inline intercom-plugin/scripts/checkForUpdate.js]:
```suggestion
  var fs = require('fs');
```

File: intercom-plugin/scripts/checkForUpdate.js [hunk 3]
```diff
@@ -19,14 +19,14 @@ function fetchUpdateInfo(context, callback) {
...
   fs.writeFile('platforms/ios/.intercom_update', JSON.stringify(updateInfo), 'utf8', function (err,data) {
     callback();
   });
 }
 
 function updateIntercomIfNeeded(context, updateInfo, callback) {
-  var exec = context.requireCordovaModule('child_process').exec;
+  var exec = context.require('child_process').exec;
```
Comment (piwel) [inline intercom-plugin/scripts/checkForUpdate.js]:
```suggestion
  var exec = require('child_process').exec;
```

File: intercom-plugin/scripts/checkForUpdate.js [hunk 4]
```diff
@@ -49,7 +49,7 @@ function updateIntercomIfNeeded(context, updateInfo, callback) {
 }
 
 function fetchLatestRelease(context, callback) {
-  var https = context.requireCordovaModule('https');
+  var https = context.require('https');
```
Comment (piwel) [inline intercom-plugin/scripts/checkForUpdate.js]:
```suggestion
  var https = require('https');
```

File: intercom-plugin/scripts/checkForUpdate.js [hunk 5]
```diff
@@ -79,7 +79,7 @@ function fetchLatestRelease(context, callback) {
 }
 
 module.exports = function(context) {
-  var Q = context.requireCordovaModule('q');
+  var Q = context.require('q');
```
Comment (piwel) [inline intercom-plugin/scripts/checkForUpdate.js]:
```suggestion
  var Q = require('q');
```

File: intercom-plugin/scripts/prepareCocoaPods.js [hunk 6]
```diff
@@ -1,7 +1,7 @@
 module.exports = function(context) {
-  var Q = context.requireCordovaModule('q');
+  var Q = context.require('q');
```
Comment (piwel) [inline intercom-plugin/scripts/prepareCocoaPods.js]:
```suggestion
  var Q = require('q');
```

File: intercom-plugin/scripts/prepareCocoaPods.js [hunk 7]
```diff
@@ -1,7 +1,7 @@
...
   var deferral = new Q.defer();
-  var exec = context.requireCordovaModule('child_process').exec;
+  var exec = context.require('child_process').exec;
```
Comment (piwel) [inline intercom-plugin/scripts/prepareCocoaPods.js]:
```suggestion
  var exec = require('child_process').exec;
```

**Comment by michaelbonner** [comment]
Hey guys! Thanks for the plugin. Is there any update on this PR? I'm currently not able to build my app using Ionic Appflow since it builds with Cordova 9

**Comment by studentIvan** [comment]
> Hey guys! Thanks for the plugin. Is there any update on this PR? I'm currently not able to build my app using Ionic Appflow since it builds with Cordova 9

seems they are not ready yet, you have to downgrade your cordova version

**Comment by jamescodesthings** [comment]
It looks like the current `master` branch would build with Cordova 9.

It just needs to be tagged and released. @DaleCantwell It looks like you were the last to release, can you help out?

**Comment by michaelbonner** [comment]
Thank you so much!
🥇

**Pull request merged**